frontend/packages/dev-console/src/components/project-access/__tests__/ProjectAccess.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { Formik } from 'formik';
+import { LoadingBox, StatusBox } from '@console/internal/components/utils';
+import ProjectAccess, { ProjectAccessProps } from '../ProjectAccess';
+
+const projectAccessProps: ProjectAccessProps = {
+  formName: 'project access',
+  namespace: 'abc',
+  roleBindings: {
+    data: [],
+    loaded: true,
+    loadError: {},
+  },
+};
+
+describe('Project Access', () => {
+  let renderProjectAccess: ShallowWrapper<ProjectAccessProps>;
+
+  it('should show the LoadingBox when role bindings are not loaded, but user has access to role bindings', () => {
+    projectAccessProps.roleBindings.loaded = false;
+    renderProjectAccess = shallow(<ProjectAccess {...projectAccessProps} />);
+    expect(renderProjectAccess.exists()).toBe(true);
```

Comment:
This assertion is unnecessary because the wrapper will always exist.
https://github.com/airbnb/enzyme/issues/1864